cli/train_postprocess_docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
I would replace train_postprocess.sh with this script. Don't have a separate dockerized version.